CHANGELOG.md

```diff
@@ -1,5 +1,72 @@
 # IPFS Cluster Changelog
 
+### v0.9.0 - 2019-02-08
+
+#### Summary
+
+IPFS Cluster version 0.9.0 comes with one big new feature, [OpenCensus](https://opencensus.io) support! This allows for the collection of distributed traces and metrics from the IPFS Cluster application as well as supporting libraries. Currently, we support the use of [Jaeger]() as the tracing backend and [Prometheus]() as the metrics backend. Support for other [OpenCensus backends](https://opencensus.io/exporters/) will be added as requested by the community.
+
+#### List of changes
+
+##### Features
+
+  * Integrate [OpenCensus](https://opencensus.io) tracing and metrics into IPFS Cluster codebase [ipfs/ipfs-cluster#486](https://github.com/ipfs/ipfs-cluster/issues/486).
+
+##### Fixes
+
+  * Fixing of `ToJSON` [ipfs/ipfs-cluster#658](https://github.com/ipfs/ipfs-cluster/issues/658) [ipfs/ipfs-cluster#659](https://github.com/ipfs/ipfs-cluster/issues/659).
```

Comment:
vertical bar

Reply:
I would maybe link this issues in the tracing entry, as they were introduced by it and fixed before the release. It's not a bug that has affected anyone on an existing release (luckily).